js/main.js

```diff
@@ -3,26 +3,119 @@
 
 var $photoInput = document.getElementById('photo-input');
 var $image = document.querySelector('img');
-var $entryForm = document.querySelector('#create-form');
+var $createForm = document.querySelector('#create-form');
+var $view = document.querySelectorAll('.view');
+var $entriesLink = document.querySelector('#entries-link');
+var $newLink = document.querySelector('#new-link');
+var $noEntry = document.querySelector('#no-entry');
 
 $photoInput.addEventListener('input', handleInput);
-$entryForm.addEventListener('submit', handleSubmit);
+$createForm.addEventListener('submit', handleSubmit);
+window.addEventListener('DOMContentLoaded', handleLoad);
+$entriesLink.addEventListener('click', viewSwap);
+$newLink.addEventListener('click', viewSwap);
+$entriesLink.addEventListener('click', dataView);
+$newLink.addEventListener('click', dataView);
+$entriesLink.addEventListener('click', noEntry);
+$newLink.addEventListener('click', noEntry);
+
+var $ul = document.querySelector('ul');
+
+function handleLoad(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var render = renderEntry(data.entries[i]);
+    $ul.appendChild(render);
+    viewSwap(data.view);
+  }
+  if (data.entries.length === 0) {
+    $noEntry.className = '';
+  } else {
+    $noEntry.className = 'hidden';
+  }
+}
 
 function handleInput(event) {
   $image.src = event.target.value;
 }
 
+function noEntry(event) {
+  if (data.view === 'entry-form') {
+    $noEntry.className = 'hidden';
+  } else if (data.entries.length === 0 && data.view === 'entries') {
+    $noEntry.className = '';
+  }
+}
```

Comment:
Instead of having a separate function for this, I would just include this work in the `viewSwap` function